src/components/features/ProgressBar.vue

```diff
@@ -0,0 +1,80 @@
+<template>
+  <div class="progress progress-striped">
+    <div class="progress__bar"></div>
+  </div>
+</template>
+
+<script>
+export default {
+  name: "ProgressBar",
+  props: ["startPercent", "config"],
+  data() {
+    return {
+      progressBar: null,
+      progressBarContainer: null
+    };
+  },
+
+  created() {
+    this.initializeProgressBar();
+  },
+
+  computed: {
+  },
+
+  methods: {
+    initializeProgressBar(){
+      setTimeout(() => {
+        this.progressBarContainer = document.querySelector(".progress")
+        this.progressBar = document.querySelector(".progress__bar")
+        this.setConfigProperties(this.config['details'])
+        this.progressTo(this.startPercent)
+      }, 200)
+    },
```

Comment:
select the progress bar container and the progress bar, set their config properties and set the initial position where the progress bar should be